R/module_nested_tabs.R

```diff
@@ -364,11 +364,11 @@ srv_teal_module.teal_module <- function(id,
 .resolve_module_datanames <- function(data, modules) {
   stopifnot("data_rv must be teal_data object." = inherits(data, "teal_data"))
   if (is.null(modules$datanames) || identical(modules$datanames, "all")) {
-    .topologically_sort_datanames(ls(teal.code::get_env(data)), teal.data::join_keys(data))
+    names(data) # TODO: Reviewer: Check that previous code does nothing.
```

Comment:
Please confirm that the previous statement did nothing

```suggestion
    names(data)
```